migrations/Version202309041508164049_taoOutcomeUi.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoOutcomeUi\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoOutcomeUi\model\ResultsViewerService;
+use oat\taoQtiTest\models\DeliveryItemTypeService;
+
+final class Version202309041508164049_taoOutcomeUi extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Migrate default item type config from oat\taoOutcomeUi\model\ResultsViewerService to oat\taoQtiTest\models\DeliveryItemTypeService';
+    }
+
+    public function up(Schema $schema): void
+    {
+        if (!$this->getServiceManager()->getContainer()->has(ResultsViewerService::SERVICE_ID)) {
+            return;
+        }
+
+        /** @var ResultsViewerService $service */
+        $service = $this->getServiceManager()->getContainer()->get(ResultsViewerService::SERVICE_ID);
+        $defaultItemType = $service->getDefaultItemType();
+
+        if (!$defaultItemType) {
+            return;
+        }
+
+        /** @var DeliveryItemTypeService $service */
+        $service = $this->getServiceManager()->getContainer()->get(DeliveryItemTypeService::SERVICE_ID);
```

Comment:
When https://github.com/oat-sa/extension-tao-testqti/pull/2398/files doesn't register `DeliveryItemTypeService` service with a migration, then this line fails and migration is aborted